tools/webassembly.py

```diff
@@ -561,6 +567,18 @@ def get_function_type(self, idx):
       func_type = self.get_function_types()[idx - self.num_imported_funcs()]
     return self.get_types()[func_type]
 
+  def get_target_features(self):
+    section = self.get_custom_section('target_features')
+    self.seek(section.offset)
+    assert self.read_string() == 'target_features'
+    features = {}
+    feature_count = self.read_byte()
+    while self.tell() < section.offset + section.size:
+      prefix = TargetFeaturePrefix(self.read_byte())
```

Comment:
What does an argument to `TargetFeaturePrefix` do here?

Reply:
It just constructs the enum with that value.